components/atoms/forms/TextInput/textInput.module.css

```diff
@@ -17,29 +17,35 @@
 
 .label {
   margin-bottom: var(--ye-spacing-em-xsmall);
-  padding: var(--input-padding-top) var(--input-padding-right) var(--input-padding-bottom) var(--input-padding-left);
+  padding: var(--input-padding-top) var(--input-padding-right)
+    var(--input-padding-bottom) var(--input-padding-left);
   line-height: var(--ye-line-height);
 }
 
 .text-input {
+  --color-body-with-opacity: #939fa480;
+
   width: 100%;
-  padding: var(--input-padding-top) var(--input-padding-right) var(--input-padding-bottom) var(--input-padding-left);
+  padding: var(--input-padding-top) var(--input-padding-right)
+    var(--input-padding-bottom) var(--input-padding-left);
   color: inherit;
   font-family: var(--ye-fontfamily-input);
-  border-color: var(--ye-color-input-border);
+  border-color: var(--color-body-with-opacity);
```

Comment:
`--ye-color-input-border` can be overriden by the consuming application, this change would hard code the color.